Auth0/WebAuthUserAgent.swift

```diff
@@ -1,7 +1,16 @@
 #if WEB_AUTH_PLATFORM
+/// Represents the external user agent used to perform q web-based operation.
```

Comment:
Typo here? "q web-based"..

Reply:
Good catch, fixed in https://github.com/auth0/Auth0.swift/pull/697/commits/25bb8faf17450b2b2ff52730b2fac1cfd3485cd8.